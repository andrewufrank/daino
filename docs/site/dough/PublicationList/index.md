---
title: index for publication list (this text should be overwritten)
author: AUF
version: publish
visibility: public
date: Jan. 4, 2019
keywords: test
abstract: index for the publication directory 
# pageTemplate: page3.yaml
# indexPage: True 
indexSort: filename
---

an index page for testing publication list

- [cadastre](/PublicationList/resources/cadastre.html)


