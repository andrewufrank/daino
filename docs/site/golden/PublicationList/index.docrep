Docrep {meta1 = MetaPage {dyFn = "/home/frank/Workspace11/ssg/docs/site/dough/PublicationList/index.md", dyLink = "PublicationList/index.md", dyLang = DLenglish, dyTitle = "index for publication list (this text should be overwritten)", dyAbstract = "index for the publication directory", dyAuthor = "FILL AUTHOR", dyDate = Just "Jan. 4, 2019", dyKeywords = "test", dyBibliography = Nothing, dyStyle = Nothing, dyReferences = Nothing, dyNoCite = Nothing, dyPublish = Nothing, dyIndexPage = True, dyIndexSort = Just "title", dyIndexEntry = IndexEntry {fn = "/home/frank/Workspace11/ssg/docs/site/dough/PublicationList/index.md", link = "PublicationList/index.md", title = "index for publication list (this text should be overwritten)", abstract = "index for the publication directory", author = "FILL AUTHOR", date = "Jan. 4, 2019", publish = Nothing, indexPage = True, dirEntries = [], fileEntries = []}}, pan1 = Pandoc (Meta {unMeta = fromList [("abstract",MetaInlines [Str "index",Space,Str "for",Space,Str "the",Space,Str "publication",Space,Str "directory"]),("date",MetaInlines [Str "Jan.",Space,Str "4,",Space,Str "2019"]),("indexPage",MetaBool True),("indexSort",MetaInlines [Str "title"]),("keywords",MetaInlines [Str "test"]),("pageTemplate",MetaInlines [Str "page3.yaml"]),("title",MetaInlines [Str "index",Space,Str "for",Space,Str "publication",Space,Str "list",Space,Str "(this",Space,Str "text",Space,Str "should",Space,Str "be",Space,Str "overwritten)"])]}) [Para [Str "an",Space,Str "index",Space,Str "page",Space,Str "for",Space,Str "testing",Space,Str "publication",Space,Str "list"]]}