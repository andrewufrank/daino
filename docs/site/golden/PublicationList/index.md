---
title: index for publication list (this text should be overwritten)
# author: AUF
date: Jan. 4, 2019
keywords: test
abstract: index for the publication directory 
pageTemplate: page3.yaml
indexPage: True 
indexSort: title
---

an index page for testing publication list



