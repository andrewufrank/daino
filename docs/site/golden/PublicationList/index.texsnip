TexSnip {snipyam = MetaPage {dyFn = "/home/frank/Workspace11/ssg/docs/site/dough/PublicationList/index.md", dyLink = "PublicationList/index.md", dyLang = DLenglish, dyTitle = "index for publication list (this text should be overwritten)", dyAbstract = "index for the publication directory", dyAuthor = "FILL AUTHOR", dyDate = Just "Jan. 4, 2019", dyKeywords = "test", dyBibliography = Nothing, dyStyle = Nothing, dyReferences = Nothing, dyNoCite = Nothing, dyPublish = Nothing, dyIndexPage = True, dyIndexSort = Just "title", dyIndexEntry = IndexEntry {fn = "/home/frank/Workspace11/ssg/docs/site/dough/PublicationList/index.md", link = "PublicationList/index.md", title = "index for publication list (this text should be overwritten)", abstract = "index for the publication directory", author = "FILL AUTHOR", date = "Jan. 4, 2019", publish = Nothing, indexPage = True, dirEntries = [], fileEntries = [IndexEntry {fn = "/home/frank/Workspace11/ssg/docs/site/dough/PublicationList/publistAF.md", link = "PublicationList/publistAF.md", title = "publistAF.md", abstract = "produce a publist for af", author = "auf", date = "2010-07-29", publish = Nothing, indexPage = False, dirEntries = [], fileEntries = []}]}}, unTexSnip = "an index page for testing publication list"}