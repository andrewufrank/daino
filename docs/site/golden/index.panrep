Panrep {panyam = MetaPage {dyFn = "/home/frank/Workspace11/ssg/docs/site/dough/index.md", dyLink = "index.md", dyLang = DLenglish, dyTitle = "XX's homepage", dyAbstract = "The index page for all.", dyAuthor = "FILL AUTHOR", dyDate = Just "2010-01-29", dyKeywords = "homepage", dyBibliography = Nothing, dyStyle = Nothing, dyReferences = Nothing, dyNoCite = Nothing, dyPublish = Nothing, dyIndexPage = False, dyIndexSort = Nothing, dyIndexEntry = IndexEntry {fn = "/home/frank/Workspace11/ssg/docs/site/dough/index.md", link = "index.md", title = "XX's homepage", abstract = "The index page for all.", author = "FILL AUTHOR", date = "2010-01-29", publish = Nothing, indexPage = False, dirEntries = [], fileEntries = []}}, panpan = Pandoc (Meta {unMeta = fromList [("abstract",MetaBlocks [Para [Str "The",Space,Str "index",Space,Str "page",Space,Str "for",Space,Str "all."]]),("date",MetaInlines [Str "2010-01-29"]),("keywords",MetaInlines [Str "homepage"]),("page-title",MetaInlines [Str "MyBlog"]),("pageTemplate",MetaInlines [Str "page3"]),("title",MetaInlines [Str "XX's",Space,Str "homepage"])]}) [Para [Str "My",Space,Str "landing",Space,Str "page",Space,Str "(TODO",Space,Str "instead",Space,Str "of",Space,Str "a",Space,Str "menu)",Space,Str "1",Space,Str ":"]]}