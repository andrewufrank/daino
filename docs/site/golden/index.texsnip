TexSnip {snipyam = MetaPage {dyFn = "/home/frank/Workspace11/ssg/docs/site/dough/index.md", dyLink = "index.md", dyLang = DLenglish, dyTitle = "XX's homepage", dyAbstract = "The index page for all.", dyAuthor = "FILL AUTHOR", dyDate = Just "2010-01-29", dyKeywords = "homepage", dyBibliography = Nothing, dyStyle = Nothing, dyReferences = Nothing, dyNoCite = Nothing, dyPublish = Nothing, dyIndexPage = False, dyIndexSort = Nothing, dyIndexEntry = IndexEntry {fn = "/home/frank/Workspace11/ssg/docs/site/dough/index.md", link = "index.md", title = "XX's homepage", abstract = "The index page for all.", author = "FILL AUTHOR", date = "2010-01-29", publish = Nothing, indexPage = False, dirEntries = [], fileEntries = []}}, unTexSnip = "My landing page (TODO instead of a menu) 1 :"}