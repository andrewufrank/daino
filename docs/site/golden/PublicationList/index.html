<!DOCTYPE html>
<!-- the master page for the pandoc templating mechanism
	needs page-title, page-title-postfix
		author date and keywords for indexing
    contentHtml, beforeContent, afterContent to wrap htmlcontent  
	uses style.css as default, other can be loaded with key css 
	-->
<html lang="DLenglish">
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />


  <style type="text/css">
      code{white-space: pre-wrap;}
      span.smallcaps{font-variant: small-caps;}
      span.underline{text-decoration: underline;}
      div.column{display: inline-block; vertical-align: top; width: 50%;}
	  </style>

	
	
          <meta name="author" content="FILL AUTHOR" />
              <meta name="dcterms.date" content="Jan. 4, 2019" />
              <meta name="keywords" content="test" />
    
	<title></title>
	
        <style>
            body {
            margin: 0;
            font-family: Arial, Helvetica, sans-serif;
            }

            .hero-image {
            background-image: url('');
            background-color: white;
            height: 200px;
            background-position: center;
            background-repeat: no-repeat;
            background-size: cover;
            position: relative;
            }

            .hero-text {
            transform: translate(-50%, -50%);
            color: white;
            text-shadow: 2px 2px 8px black;
            font-weight: bold;
            vertical-align: top;
            }
        </style>
    </head>

    <body>
    <!-- AF - my template for pandoc page (from the default) page3.dtpl -->
        

    <section class="hero">
        <div class="hero-image">
            
                <h1></h1>
                <h3></h3>
        <div/>
    </section>
    <section class="header">

            <section class="menu">
		<ul>
            			</ul>
    </section>
    <section>
                <header>
        <h1 class="title">index for publication list (this text should be overwritten)</h1>
        
                <p class="author">FILL AUTHOR</p>
                </header>
         
    </section>

    <section class=menu2>
		<ul>
            					<li>
				    <a href=PublicationList/index.HTML><em>index for publication list (this text should be overwritten) </em></a>
				    <br>index for the publication directory		     
				    <span class=tiny>FILL AUTHOR "Jan. 4, 2019" 
            PublicationList/index.HTML Nothing
                    </span>
                    
    			</li>
            						<li>
				    <a href=><em>--- subdir --- </em></a>
				    		     
				    <span class=tiny>  
             
                    </span>
                    
    			</li>
            						<li>
				    <a href=PublicationList/publistAF.HTML><em>publistAF.md </em></a>
				    <br>produce a publist for af		     
				    <span class=tiny>auf "2010-07-29" 
            PublicationList/publistAF.HTML Nothing
                    </span>
                    
    			</li>
            				</ul>
    </section>

        
<!-- blog text content section -->
    
    abstractindex for the publication directory 
    content-from-panrep-blocks<p>an index page for testing publication list</p>
    


        <p class=tiny>
        Produced with SGG  on  from  with master4.dtpl.
        </p>

            </body>


</html>
